tools/maxquant/maxquant.xml

```diff
@@ -23,9 +23,7 @@
     --version='@VERSION@'
     --fasta_files='$fasta_files'
     #if $mode.choose_mode == "interactive"
-        #if $mode.fractions.set_fractions=="true"
-            --exp_design='$mode.fractions.template'
-        #end if
+        --exp_design='$mode.template'
```

Comment:
Please note that $mode.template can be None ... in the case the user does not provide the file. So the if is probably needed.